cl/users/email_handlers.py

```diff
@@ -417,7 +403,41 @@ def is_message_stored(message_id: str) -> tuple[bool, int | None]:
     return False, None
 
 
-from cl.users.tasks import send_failed_email
+def schedule_failed_email(recipient_email: str) -> None:
+    """Schedule recipient's waiting failed emails after receiving a delivery
```

Comment:
This docstring is no longer correct since we don't rely on delivery notifications anymore.

Reply:
In fact, do a grep for `delivery notification` and you'll find a bunch more examples that need updating.

Reply:
Thanks, I've looked for `delivery notification` and updated it.